src/main/java/com/redhat/red/build/koji/model/xmlrpc/KojiBuildInfo.java

```diff
@@ -112,7 +114,7 @@
     private String source;
 
     @JsonIgnore
-    private String typeName;
+    private List<String> typeName; // a build may contain more than one types, e.g., maven and rpm.
```

Comment:
Since there are multiple types, this field name and associated method names should be made plural (end in 's').

Additionally, I had liked the name 'types' better than 'typeNames'. However, we can't store `KojiBuildTypeInfo` instead of `String` as all of this information except for the name is already being merged into `KojiBuildInfo` from `KojiClient`, so maybe the existing name is fine.

Reply:
Changed to typeNames.